config/initializers/environment_check.rb

```diff
@@ -13,6 +13,16 @@ def mandatory_secret!(env, value)
         "mandatory in production because it's needed for the generation of secrets."
 end
 
+# Database Environment Variables have changed.
+if ENV["PORTUS_PRODUCTION_HOST"].present? && ENV["PORTUS_DB_HOST"].blank? ||
+    ENV["PORTUS_PRODUCTION_USERNAME"].present? && ENV["PORTUS_DB_USERNAME"].blank? ||
+    ENV["PORTUS_PRODUCTION_PASSWORD"].present? && ENV["PORTUS_DB_PASSWORD"].blank? ||
+    ENV["PORTUS_PRODUCTION_DATABASE"].present? && ENV["PORTUS_DB_DATABASE"].blank?
+  raise Portus::DeprecationError, "The database configuration has been extended to also support " \
+  "PostgreSQL as the database. With this change the Environment Variable Names have also been " \
+  "changed. Please consolidate config/database.yml for the new names."
```

Comment:
I would avoid explaining the PostgreSQL part, since we've also done it to simplify the naming of these environment variables. I'd simply instruct that now environment variables that started with `PORTUS_#{Rails.env.upcase}_`, now they should start simply with `PORTUS_DB_`.

Reply:
according to af1b8b6ca7 staging, development and test where previously hardcoded.